phoenix-core/src/main/java/org/apache/phoenix/expression/function/ArrayToJsonFunction.java

```diff
@@ -0,0 +1,87 @@
+package org.apache.phoenix.expression.function;
+
+import org.apache.hadoop.hbase.io.ImmutableBytesWritable;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.apache.phoenix.exception.SQLExceptionCode;
+import org.apache.phoenix.exception.SQLExceptionInfo;
+import org.apache.phoenix.expression.Expression;
+import org.apache.phoenix.parse.FunctionParseNode;
+import org.apache.phoenix.schema.IllegalDataException;
+import org.apache.phoenix.schema.SortOrder;
+import org.apache.phoenix.schema.json.PhoenixJson;
+import org.apache.phoenix.schema.tuple.Tuple;
+import org.apache.phoenix.schema.types.*;
+import org.apache.phoenix.util.ByteUtil;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.List;
+
+
+@FunctionParseNode.BuiltInFunction(name=ArrayToJsonFunction.NAME,  args={
+        @FunctionParseNode.Argument(allowedTypes={PVarchar.class})} )
+public class ArrayToJsonFunction extends ScalarFunction {
+    public static final String NAME = "Array_To_Json";
```

Comment:
Please use only capital letters for function name. I guess thats the general convention we are following for other functions  though I dont know whether it has any functional significance or not.